### PR TITLE
Add Chromatic

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective

Adds Chromatic for Component Library PR reviews. Also added a11y addon to stprybook, made storybook runable without web and added colors story.

## Before you submit

- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+name: Chromatic
+
+on: push
+
```
Comment (joseph-flinn) [inline .github/workflows/chromatic.yml:5]: Does this need to run on every change to jslib? If it just needs to run on changes to `./components`, can we add a `paths` block?

Comment (Hinton) [reply .github/workflows/chromatic.yml]: We should be safe to only run it when components change.

**Comment by joseph-flinn** [comment]
@Hinton Looks good from the DevOps point of view

**Review by vincentsalucci — APPROVED** [review]
I don't really speak github workflow - so I'll leave that to @joseph-flinn

**Pull request merged**